src/views/Home/index.tsx

```diff
@@ -1,36 +1,98 @@
-import { Box, Link, Typography } from '@mui/material'
+import { Box, Fade, Link, Typography } from '@mui/material'
 import { Stack } from '@mui/system'
-
+import { FileUploadOutlined as UploadIcon } from '@mui/icons-material'
 import { useIntl } from 'react-intl'
+import { DropzoneInputProps, DropzoneRootProps, useDropzone } from 'react-dropzone'
+import { useCallback, useState } from 'react'
 import { AddMapButton } from '../../components/AddMapButton'
 import { AuthorisedLayout } from '../../layouts/Authorised'
-import { Img } from './styles'
+import { Img, Overlay } from './styles'
 import { messages as msgs } from './messages'
+import { unzip } from '../../helpers/file'
+import { Loader } from '../../components/Loader'
+import { useCreateMap } from '../../hooks/useCreateMap'
+
+export const HomeView = () => {
+  const {
+    createMap,
+    progress: { completed, loading, error, done },
+  } = useCreateMap()
+
+  const onDrop = useCallback(async (acceptedFiles: File[]) => {
+    if (!acceptedFiles.length || !acceptedFiles[0].name.match(/.mapeomap$/))
+      return console.log('invalid file', acceptedFiles[0])
+    const unzippedFiles = await unzip(acceptedFiles[0])
```

Comment:
There is a slight delay between when the user adds a map and when the `loading` state starts. I believe this is going to be due to the unzip. Maybe we should move `unzip` to the hook?